docs/root/intro/arch_overview/http/http_routing.rst

```diff
@@ -79,6 +78,10 @@ If the "addr" header value is "foo=1;x-foo-key=127.0.0.1;x-bar-key=1.1.1.1", the
 
 .. _arch_overview_http_routing_route_table:
 
+For a key to match a ScopedRouteConfiguration, the number of fragments in the computed key has to match that of the ScopedRouteConfiguration.
```

Comment:
Can you use RST `envoy_msg...` links for each of these messages back to the API docs?